frontend/src/app/components/ExperimentPage.js

```diff
@@ -338,6 +360,15 @@ export default class ExperimentPage extends React.Component {
     }, 5000);
   }
 
+  clickUpgradeNotice() {
```

Comment:
I think we want this receive the `evt` argument, prevent the default action, and use 

`outboundURL: evt.target.attributes.href`

to prevent a race condition on this request completing and the user navigating away from the page.